src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -417,6 +417,65 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         execCond("docker rmi -f ${id}") || true
     }
 
+    def 'outputs from default distribution locations are added to the docker context'() {
```

Comment:
So you are saying I cannot use the default artifact location of the official Gradle distribution plugin?  Then I have to either hack my Dockerfile or create a custom tar task for your plugin to even work.